ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -534,6 +551,17 @@ var _ = Context("Inside the default namespace", func() {
 			Eventually(
 				getClusterState(ctx, namespace, rayCluster.Name),
 				time.Second*3, time.Millisecond*500).Should(Equal(rayv1.Suspended))
+
+			if withConditionEnabled {
+				Eventually(
+					func() rayv1.RayClusterConditionType {
+						if err := getResourceFunc(ctx, client.ObjectKey{Name: rayCluster.Name, Namespace: namespace}, rayCluster)(); err != nil {
+							return ""
+						}
+						return utils.FindRayClusterSuspendStatus(rayCluster)
+					},
+					time.Second*10, time.Millisecond*1000).Should(Equal(rayv1.RayClusterSuspended))
```

Comment:
nit: just make the parameters consistent with others
```suggestion
					time.Second*3, time.Millisecond*500).Should(Equal(rayv1.RayClusterSuspended))
```